### ISSUE TITLE
Product version restore throws error

### ISSUE THREAD

**Issue by tsarma** [isotope/core#1506]
Contao 3.5.2
isotope 2.3.0-RC2	5
dcawizard 2.3.0	12
dc_multilingual 2.0.0	15
haste 4.8.0	1
MultiColumnWizard 3.3.3	3
NamespaceClassLoader 1.0.2	21
notification_center 1.3.0-RC1	5
tablelookupwizard 2.0.1	23

```
Fatal error: Uncaught exception Exception with message Query error: Unknown column 'tier_keys' in 'field list' (INSERT INTO tl_iso_product_price (id, pid, tstamp, tax_class, config_id, member_group, start, stop, tier_keys, tier_values) VALUES ('2', '1', '1439294544', '0', '0', '0', '', '', '1', '0.00')) thrown in system/modules/core/library/Contao/Database/Statement.php on line 295
#0 system/modules/core/library/Contao/Database/Statement.php(264): Contao\Database\Statement->query()
#1 system/modules/isotope/library/Isotope/Backend/Product/Price.php(70): Contao\Database\Statement->execute()
#2 system/modules/core/classes/Versions.php(287): Isotope\Backend\Product\Price->restoreVersion('1', 'tl_iso_product', Array, '37')
#3 system/modules/isotope/drivers/DC_ProductData.php(424): Contao\Versions->restore('37')
#4 system/modules/core/classes/Backend.php(639): DC_ProductData->edit()
#5 system/modules/core/controllers/BackendMain.php(131): Contao\Backend->getBackendModule('iso_products')
#6 contao/main.php(20): Contao\BackendMain->run()
#7 {main}
```

**Comment by aschempp** [comment]
Did you add an attribute `tier_keys` ?

**Comment by Toflar** [comment]
I think we should have an "attribute name black list" anyway. I recall there was an issue with naming it `date` as well ;) So we could list all mysql reserved keywords plus all the others we can think of and extend it over time :)

**Comment by aschempp** [comment]
There was such a list in 1.* but it was replaced by the «systemColumn» attribute. The mentioned column name shouldn’t be a problem though.

**Comment by tsarma** [comment]
No I did not add attribute `tier_keys`
![bildschirmfoto 2015-08-18 um 14 25 40](https://cloud.githubusercontent.com/assets/683894/9329860/5da057e6-45b5-11e5-81be-286b09dcd75c.png)

**Comment by tsarma** [comment]
It worked only if "Advanced pricing" is enabled under product type setting!

![screen shot 2015-08-24 at 12 07 47](https://cloud.githubusercontent.com/assets/683894/9437734/cd988652-4a58-11e5-80bc-06e99ec5f8ce.png)

**Comment by aschempp** [comment]
Did you have advanced prices enabled when the product was deleted?

**Comment by tsarma** [comment]
Advance prices wasn't enabled when the problem occurred. Also the product wasn't deleted (It was the only product when I was testing). It was edited and saved few times to play with version restore.

**Comment by aschempp** [comment]
Fixed in 07c9d5584f0dc96632f78ec56739b82e24eec67d
